CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Another example [here](https://co-pilot.dev/changelog)
 
 - Update PostgreSQL image to timescale/timescaledb-ha:pg16 to match railway postgres image ([#210](https://github.com/chingu-x/chingu-dashboard-be/pull/210))
 - refactored e2e tests to use absolute import paths ([#207](https://github.com/chingu-x/chingu-dashboard-be/pull/207))
+- refactored unit tests to use absolute import paths ([#215](https://github.com/chingu-x/chingu-dashboard-be/pull/215))
```

Comment:
ideally we want to have these in order of completion / merge, since #206 is merged, we should move this below it